README.md

```diff
@@ -1,13 +1,11 @@
 # AWS managed OpenTelemetry Lambda Layers
 
 As a downstream Repo of [opentelemetry-lambda](https://github.com/open-telemetry/opentelemetry-lambda), ___aws-otel-lambda___ publishes AWS managed OpenTelemetry Lambda layers that are preconfigured for use with AWS services and bundle the reduced AWS Collector. Users can onboard to OpenTelemetry in their existing Lambda functions by adding these ready-made layers directly.
-- Python3.8 layer [aws-otel-python38-ver-1-5-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-python) contains OpenTelemetry Python `v1.5.0` with Contrib `v0.24b0`
-- Nodejs layer [aws-otel-nodejs-ver-0-24-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) contains OpenTelemetry JavaScript `v0.24.0` with Contrib `v0.24.0`
-- Java-Wrapper layer [aws-otel-java-wrapper-ver-1-5-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java) contains OpenTelemetry Java `v1.5.0`
-- Java-Agent layer [aws-otel-java-agent-ver-1-5-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java-auto-instr) contains AWS OpenTelemetry Java instrumentation `v1.5.0`
-- Collector layer [aws-otel-collector-ver-0-33-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-dotnet) contains ADOT Collector for Lambda `v0.12.0`
-
-
+- Python3.8 layer [aws-otel-python38-ver-1-5-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-python) contains AWS Python Extension `v1.0.1` which uses OpenTelemetry Python `v1.5.0`
+- Nodejs layer [aws-otel-nodejs-ver-0-25-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) contains OpenTelemetry JavaScript `v0.25.0` with AWS Lambda Instrumentation `v0.26.0`
```

Comment:
Just curious because doesn't NodeJS also have a `propagator` and `id-generator`? In that case should we mention it as an "AWS JS Extension" to be consistent with Python above?

I guess you can argue that they are included in reference to `OpenTelemetry JavaScript v0.24.0`.

Reply:
Yeah I just decided not to include the ID generator/propagator versions because I don't anticipate them really changing or having major new features in future versions unlike the Lambda instrumentation. JS also doesn't have "extensions" like python so there's no easy way of grouping all of them.

Reply:
That makes sense! Sounds good to me!